examples/preprocessing/plot_scaling_importance.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""
+=========================================================
+Importance of Feature Scaling
+=========================================================
+
+Features scaling though standardization (or Z-score normalization)
+can be an importance preprocessing step for many machine learning
+algorithms. Standardization involves rescaling the features such
+that they’ll have the properties of a standard normal distribution
+with a mean of zero and a standard deviation of one.
+
+While many algorithms (such as SVM, K-nearest neighbors and logistic
+regression) require features to be normalized, intuitively we can
+think of Principle Component Analysis (PCA) as being a prime example
+of when normalization is important. In PCA we are interested in the
+components that maximize the variance. If there exists components
+(e.g human height) that vary less then other components (e.g human
+weight) because of their respective scales (meters vs. kilos) it can
+be seen how not scaling the features would cause PCA to determine that
+the direction of maximal variance more closely corresponds with the
+‘weight’ axis. As a change in height of one meter can be considered much
+more important than the change in weight of one kilogram, it is easily
+seen that this determination is incorrect. In the case of PCA, scaling
+features using normalization is preferred over using min-max scaling as
+the primary components are computed using the correlation matrix as opposed
+to the covariance matrix.
+
+In order to illustrate this in an example, PCA will be performed on a dataset
+which has been standardized using StandardScalerand a copy which has remained
+untouched. The results with be visualized and a clear difference noted.
+
+The results will then be used to train a naive Bayes classifier, and a clear
+difference the prediction accuracies will be observed.
+
+"""
+from __future__ import print_function
+print(__doc__)
+
+from sklearn.cross_validation import train_test_split
+from sklearn import preprocessing
+from sklearn.decomposition import PCA
+from sklearn.naive_bayes import GaussianNB
+from sklearn import metrics
+import matplotlib.pyplot as plt
+from sklearn.datasets import load_wine
+
+
+# Code source: Tyler Lanigan <tylerlanigan@gmail.com>
+# 			   Sebastian Raschka <mail@sebastianraschka.com>
+
+# License: BSD 3 clause
+
+
+# Contants
```

Comment:
Con**s**tants

To be honest this is the kind of comments I would remove, since it doesn't bring much added value